packages/mobile/src/components/countdown/components/box/box.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+import { View, Text } from '~/components/components';
+import { useCustomTheme } from '~/hooks/hooks';
+import { globalStyles } from '~/styles/styles';
+import { BoxType } from '../../common/enums/enums';
+import { styles } from './styles';
+
+type Props = {
+  timeName: string;
+  timeValue: string;
+  boxPosition?: BoxType;
+};
+
+const Box: FC<Props> = ({ timeName, timeValue, boxPosition }) => {
```

Comment:
Let's give some reasonable names.
```suggestion
const CountdownSegment: FC<Props> = ({ timeName, timeValue, boxPosition }) => {
```

Reply:
Changed